packages/loading-indicator/src/BlobLoader/BlobLoader.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import BlobLoader from './BlobLoader';
+
+describe('packages/loading-indicator/blob-loader', () => {
+  describe('a11y', () => {
+    test('does not have basic accessibility issues', async () => {
+      const { container } = render(<BlobLoader />);
+      const results = await axe(container);
+      expect(results).toHaveNoViolations();
+    });
+  });
+  describe('description prop', () => {
```

Comment:
saw that you brought in jest-canvas, is there not another way to test the blob?

Reply:
It seems like the best package for this